ci/workflows/pull-request.yml

```diff
@@ -790,6 +790,38 @@ steps:
     timeout_in_minutes: 30
     retry: *auto-retry
 
+  - label: "end-to-end test (parallel, postgres backend)"
```

Comment:
Shall we just run e2e-test against postgres as the default option?
For the others we can run in `main-cron`

Reply:
Otherwise it may be expensive to run all combination, across all our e2e test types.

Reply:
Nvm just saw the labeller workflow.